### PR TITLE
Seeding Log Creation Confirmation

### LINKED ISSUE #285: Seeding Log Creation Confirmation
When a new seeding log is submitted via the "Seeding Report" sub-tab of the "Field Kit", a confirm dialog comes up.  This makes it apparent to the user that the log was created. We should think about the default behavior and button options on this dialog.  For example:
* Should the dialog be confirm - Create seeding log? [OK | Cancel], then the user could cancel the creation if the submit button was clicked/tapped by accident.
* What should be the default behavior after OK or Cancel is clicked?
  * For Cancel seems it should return to the seeding log form with the current values.
  * For OK, should it go back to a blank seeing input form so another entry could be created quickly?  What is the common workflow? Do they typically create multiple seeding logs in a row?

Would be a good conversation to have with Matt.   

Whatever is ultimately implemented here should also have Cypress tests that check the behavior.

### PR DESCRIPTION
**Pull request by Batese2001**
__Pull Request Description__

Resolves #285 
It will:
- create logs on a dialogue box confirmation
- alert the user that this was successful with some kind of pop up
- reset all values to their initial state on confirmation
- leave them as they were on rejection
- include tests for these

---
__Licensing Certification__

FarmData2 is a [Free Cultural Work](https://freedomdefined.org/Definition) and all accepted contributions are licensed as described in the LICENSE.md file. This requires that the contributor holds the rights to do so. By submitting this pull request __I certify that I satisfy the terms of the [Developer Certificate of Origin](https://developercertificate.org/)__ for its contents.

### CODE REVIEW

**Comment by Batese2001** [comment]
This last commit does not reset the dropdowns. Do we want the dropdowns to be reset as well?

File: farmdata2_modules/fd2_tabs/fd2_field_kit/seedingInput.html [hunk 1]
```diff
@@ -83,8 +101,36 @@ <h1 style='text-align:center;'>Seeding Input Log</h1>
...
+                stopAlert(){
+                    this.alertVal = false
+                },
+                clearForm(){
+                    this.cropArray.push('')
+                    this.areaFilter.push('')
+                    console.log(this.cropArray)
+                    console.log(this.areaFilter)
+                    this.selectedDate = dayjs().format('YYYY-MM-DD').toString()
+                    this.selectedCrop = "ARUGULA"
```
Comment (braughtg) [inline farmdata2_modules/fd2_tabs/fd2_field_kit/seedingInput.html:116]: Should the selectedCrop be set to '' here in the same way the selectedArea is below?

**Comment by braughtg** [comment]
WIth #350 merged the issues with this PR not resetting dropdown elements should be fixed.  There will still be issues with resetting of date elements that will require #351 and #352 to be resolved.

**Comment by braughtg** [comment]
* Superseded by #285.

**Pull request closed without merging**